Assets/Scripts/Control&Input/Peer2PeerTransport.cs

```diff
@@ -325,24 +392,26 @@ public override void OnServerChangeScene(string newSceneName)
             AddAiPlayers();
         }
 
-        UpdatePlayerDetails();
 
         switch (newSceneName)
         {
             case Scenes.Bidding:
+                UpdatePlayerDetailsAfterShootingRound();
                 NetworkServer.ReplaceHandler<SpawnPlayerMessage>(OnSpawnBiddingPlayer);
                 break;
             case Scenes.Menu:
                 NetworkServer.RegisterHandler<PlayerConnectedMessage>(OnSpawnPlayerInput);
                 break;
             default:
+                // From auction, so no need to update *here*
                 NetworkServer.ReplaceHandler<SpawnPlayerMessage>(OnSpawnFPSPlayer);
                 break;
         }
 
         base.OnServerChangeScene(newSceneName);
     }
 
+    // TODO move to ItemSelectManager (?)
     private void UpdateLoadout()
```

Comment:
I assume this might be tweaked in #661, so I'll ignore this if that's the case

Reply:
I intend to do this thing, likely in #661